hack/generator/pkg/codegen/pipeline/apply_export_filters.go

```diff
@@ -15,10 +15,12 @@ import (
 	"github.com/Azure/azure-service-operator/hack/generator/pkg/config"
 )
 
+const ApplyExportFiltersStageID = "filterTypes"
+
 // ApplyExportFilters creates a Stage to reduce our set of types for export
 func ApplyExportFilters(configuration *config.Configuration) Stage {
 	return MakeLegacyStage(
```

Comment:
Any chance you've got time to change this to use `MakeStage()` instead? #minor

Reply:
Changed